api_endpoints.py

```diff
@@ -4,10 +4,17 @@
 import ast
 import csv
 import os
+from time import sleep
+import sys
 
 app = Flask(__name__)
 api = Api(app)
 
+SLEEP_TIME = int(1)
+
+if(len(sys.argv)>=2):
+    SLEEP_TIME = int(sys.argv[1])
```

Comment:
We can now run:

`python api_endpoints.py 15
`

So we can check `POST` requests that are taking `15 `seconds at least.

It is good for testing background flow on our client